src/main/java/org/folio/util/TokenCookieParser.java

```diff
@@ -0,0 +1,58 @@
+package org.folio.util;
+
+import org.apache.commons.lang3.StringUtils;
+import org.folio.rest.impl.LoginAPI;
+
+public class TokenCookieParser {
+  public String getAccessToken() {
+    return accessToken;
+  }
+  private String accessToken;
+
+  public String getRefreshToken() {
+    return refreshToken;
+  }
+  private String refreshToken;
+
+  public TokenCookieParser(String cookieHeader) {
+    accessToken = getCookieValue(cookieHeader, LoginAPI.ACCESS_TOKEN);
+    refreshToken = getCookieValue(cookieHeader, LoginAPI.REFRESH_TOKEN);
+  }
+
+  private String getCookieValue(String cookieHeader, String name) {
+    String headerStripped = StringUtils.strip(cookieHeader, "\t");
```

Comment:
Can you use an existing parser like ServerCookieDecoder.STRICT.decodeAll(cookieHeader)? This line doesn't handle "\t \t foo".
https://netty.io/4.1/api/io/netty/handler/codec/http/cookie/ServerCookieDecoder.html#decodeAll-java.lang.String-

Reply:
Added netty's `ServerCookieDecoder` into the mix.